source/shared/cpp/ObjectModel/MarkDownParser.cpp

```diff
@@ -50,7 +56,8 @@ void MarkDownParser::ParseBlock()
 std::string MarkDownParser::EscapeText()
 {
     std::string escaped;
-
+    unsigned int unescapedCounts = 0;
```

Comment:
I think something like `nonEscapedCounts` would be a better variable name -- "unescaped" sounds like an action has occurred. :)

Reply:
thanks :-)

---
In reply to: [215793995](https://github.com/Microsoft/AdaptiveCards/pull/1865#discussion_r215793995) [](ancestors = 215793995)